libs/client/dashboard/src/lib/pages/tags-management/tags-management.component.ts

```diff
@@ -27,7 +27,12 @@ export class TagsManagementComponent implements OnInit {
     }
 
     addChild(parentId: string) {
-        this.dialog.open(ChildTagFormComponent, { data: { parentId: parentId }});
+        const alertData: PopupModel = {
+            message: 'Currently disabled',
+            confirm: true,
+        };
+        const dialogRef = this.dialog.open(PopupComponent, { data: alertData });
+        // this.dialog.open(ChildTagFormComponent, { data: { parentId: parentId }});
```

Comment:
What's the commented-out line for?

Reply:
That triggers the flow for adding a tag as the child of the given tag. I've disabled it for now.